### PR TITLE
Frontend text fixes

### PR DESCRIPTION
**Pull request by drewpotter**
This is the initial work to hardcode some temporary text fixes using hardcoded urls rather than using Rails routes.

### CODE REVIEW

**Comment by WKocur** [comment]
Screenshot 2019-06-20 at 16 30 10
Should this support link go to the admin support page or the public one? Current links to the public one.

File: app/views/surveys/show.html.erb [hunk 1]
```diff
@@ -23,9 +23,9 @@
 
   <%= link_to "Take the survey", new_survey_response_path(@survey), class: 'button' %>
   <%= link_to 'Edit', edit_survey_path(@survey), class: 'button' %>
-  <%= link_to 'Back', surveys_path, class: 'button' %>
-  <%= render partial: "surveys/export_form", locals: { survey: @survey } %>
-
+  <p>
```
Comment (WKocur) [inline app/views/surveys/show.html.erb]: This button could do with a little bit of margin-top (and possibly remove margin-left). Just check that you don't change the styling on the other use of the popup on the index page.

**Comment by WKocur** [comment]
> Good question @WKocur I wasn't aware that I had changed it... 🤔

Looks like it just needs switching to `admin/support` in the yml content (`resources.content` line 149)

File: app/views/surveys/show.html.erb (hunk 1 above)
Comment (drewpotter) [reply app/views/surveys/show.html.erb]: Thanks!

**Comment by drewpotter** [comment]
Thanks @WKocur !

**Comment by lucacug** [comment]
> Screenshot 2019-06-20 at 16 30 10
> Should this support link go to the admin support page or the public one? Current links to the public one.

@WKocur Sorry Will this should point to the public or to the admin support page then ??
Currently it's still pointing to the public one

**Comment by rogeringle** [comment]
@lucacug – should point to the admin side one 👍

**Pull request merged**